packages/contracts/hardhat.config.ts

```diff
@@ -23,15 +22,7 @@ import { ChainSymobls, getTokenBySymbol } from '@eonian/upgradeable'
 import { binanceSmartChainFork, ethereumFork } from './hardhat/forks'
 import { ProvidersContracts, providers } from './hardhat/providers'
 
-import './hardhat/tasks/start-hardhat-node.ts'
-
-task('accounts', 'Prints the list of accounts', async (taskArgs, hre) => {
-  const accounts = await hre.ethers.getSigners()
-
-  for (const account of accounts) {
-    console.log(account.address)
-  }
-})
+import './hardhat/tasks'
```

Comment:
If it must be after all imports then maybe add comment?
```suggestion
// must be after all imports
import './hardhat/tasks'
```

Reply:
no, not necessarely, it should be below dotenv, but we already have the comment for it